Model tests and factories

nunomsantos opened the pull request:
Ironically enough, one of these new tests is still [failing](https://travis-ci.org/engwebUM/parkyourcar/builds/62840800)  – but it should pass after rebasing from PR #29.

Further discussion needed on the necessity of the three tests being temporarily skipped.

fredCarvalho commented on spec/factories/attachments.rb:
```diff
@@ -0,0 +1,10 @@
+require 'faker'
```
Maybe you don't need this gem on this file

fredCarvalho commented on spec/factories/bookings.rb:
```diff
@@ -0,0 +1,25 @@
+require 'faker'
```
And here faker is not necessary

fredCarvalho commented on spec/models/user_spec.rb:
```diff
@@ -0,0 +1,59 @@
+    end
+  end
+
+  describe '#photo' do
```
Don't forget to implement these tests

fredCarvalho commented:
After the corrections and rebasing to pass the tests you can merge.

nunomsantos commented:
Thanks, @fredCarvalho, I've addressed most of your comments.

There are still some tests being temporarily skipped because they need further discussion. I think this discussion can be done later and that we should merge this branch as it is right now.

fredCarvalho commented:
Ok. The tests you've done are very useful, so you can merge now :+1:
